apiserver/DEVELOPMENT.md

```diff
@@ -7,8 +7,12 @@ This guide covers the purpose, requirements, and deployment of the Kuberay API S
 | Software | Version  |                                                            Link |
 | :------- | :------: | ------------------------------------------------------------: |
 | kubectl  | v1.18.3+ | [Download](https://kubernetes.io/docs/tasks/tools/install-kubectl/) |
-| Go       |  v1.13+  |                            [Download](https://golang.org/dl/) |
+| Go       |  v1.17   |                            [Download](https://golang.org/dl/) |
```

Comment:
Is it strictly 1.17 or 1.17+

Reply:
For now strictly 1.17 as it the version defined in the `go.mod`. I have ran things with 1.18, no issues found.